packages/riipen-ui/src/components/Menu.jsx

```diff
@@ -104,9 +104,8 @@ class Menu extends React.Component {
   }
 
   handleChange = (idx, event) => {
-    const { onChange, closeOnClick } = this.props;
+    const { onChange } = this.props;
     if (onChange) onChange(idx, event);
-    if (closeOnClick && event && event.type === "click") this.handleClose(idx);
   };
 
```

Comment:
I think this is the only place that closeOnClick is used, 

are we saying that it is now the job of the parent component to close to menu onChange if they want to? I just dont fully know the implications for the dropdown menus on the Web app

Reply:
Good catch! This was actually from me screwing around previously and forgot to fix. 

I ended up just re-ordering the close and the on change triggers. The issue was that `onChange` may cause the component to unmount, then running `handleClose` would screw up.